code/modules/mob/living/carbon/xenomorph/castes/king/abilities_king.dm

```diff
@@ -115,6 +139,88 @@
 		return ..()
 	return TRUE
 
+
+/datum/action/xeno_action/activable/gravity_crush_lesser
+	name = "Lesser Gravity Crush"
+	action_icon_state = "fortify"
+	mechanics_text = "Increases the localized gravity in an area; weakening targets"
+	ability_name = "Lesser Gravity Crush"
+	plasma_cost = 50
+	cooldown_timer = 5 SECONDS //Much lower
+	/// How far can we use gravity crush
+	var/king_crush_dist = 8 //Further range, smaller crush
+	/// A list of all things that had a fliter applied
+	var/list/filters_applied = list()
+	var/cooldowntext = "<span class='warning'>Our psychic aura restores itself. We are ready to lesser gravity crush again.</span>"
```

Comment:
Just want to state for the record that if we had any sort of established codebase, this wouldn't pass. Copy+pasting an entire 79 line datum with only a few minor changes when it would be far better as a subtype.
That said, having it as a subtype doesn't work currently, and this is still very early days. If it works, don't fix it.

Maybe it'll be refactored in the future, but this is fine for now. 👍

Reply:
oh I 100% know and this was shitcode to get it working and acceptable for ingame use. I'll actually make these alternate action buttons like the queen gets involving resin spit in time.